api.php

```diff
@@ -0,0 +1,38 @@
+<?php
+include_once 'db.php';
+
+if (date("N") == 6) {
+    $filename = "../data/" . date("Y-m-d", time() + 60 * 60 * 24 * 2) . ".db";
+    $humanReadableDay = "Montag";
+} else if (date("N") == 7) {
+    $filename = "../data/" . date("Y-m-d", time() + 60 * 60 * 24) . ".db";
+    $humanReadableDay = "Montag";
+} else if (date("N") == 5 && date("H") >= 15) {
+    $filename = "../data/" . date("Y-m-d", time() + 60 * 60 * 24 * 3) . ".db";
+    $humanReadableDay = "Montag";
+} else if (date("H") >= 15) {
+    $filename = "../data/" . date("Y-m-d", time() + 60 * 60 * 24) . ".db";
+    $humanReadableDay = "Morgen";
+} else {
+    $filename = "../data/" . date("Y-m-d") . ".db";
+    $humanReadableDay = "Heute";
+}
```

Comment:
As this code is now double, I think it should be moved to db.php, returning an array with `filename` and `humanReadableDay`